README.md

```diff
@@ -24,51 +24,53 @@ more improvements. Browse the [list of TIPs](#list-of-tips) below with the _Star
 
 ## List of TIPs
 
- - Last updated: 2022-05-20
+ - Last updated: 2023-10-19
  - The _Status_ of a TIP reflects its current state with respect to its progression to being supported on the IOTA mainnet.
    - `Draft` TIPs are work in progress. They may or may not have a working implementation on a testnet.
    - `Proposed` TIPs are demonstrated to have a working implementation. These TIPs are supported on Shimmer, the staging network of IOTA.
    - `Active` TIPs are supported on the IOTA mainnet.
+   - `Replaced` TIPs have been replaced by a newer TIP.
+   - `Obsolete` TIPs are no longer in use.
```

Comment:
Missing `Withdrawn`

Reply:
lemme just link here the figure from TIP-1 https://github.com/iotaledger/tips/raw/main/tips/TIP-0001/process.svg